src/fdcache.cpp

```diff
@@ -336,8 +336,12 @@ void PageList::Clear()
 bool PageList::Init(off_t size, bool is_loaded, bool is_modified)
 {
   Clear();
-  fdpage page(0, size, is_loaded, is_modified);
-  pages.push_back(page);
+  if(0 < size){
+    fdpage page(0, size, is_loaded, is_modified);
+    pages.push_back(page);
+  }
+  Dump();
```

Comment:
Needed?

Reply:
Without this judgment, I was creating a useless fdpage object with offset 0 and size 0.
It doesn't lead to any trouble, but I think it is better not to create it.

Reply:
Sorry I mean is the call to `Dump` needed?

Reply:
The call to `Dump` only logs the list so I don't think it's necessary.

Reply:
I'm sorry for my fault, this line is for my debugging.
It will be removed soon.

Reply:
I fixed this, and rebase this PR from the current master.